README.md

```diff
@@ -1,12 +1,60 @@
 # [qri](http://qri.io)
 
-[![Qri](https://img.shields.io/badge/made%20by-qri-magenta.svg?style=flat-square)](https://qri.io)
-[![GoDoc](https://godoc.org/github.com/qri-io/qri?status.svg)](http://godoc.org/github.com/qri-io/qri)
-[![License](https://img.shields.io/github/license/qri-io/qri.svg?style=flat-square)](./LICENSE)
-[![Codecov](https://img.shields.io/codecov/c/github/qri-io/qri.svg?style=flat-square)](https://codecov.io/gh/qri-io/qri)
-[![CI](https://img.shields.io/circleci/project/github/qri-io/qri.svg?style=flat-square)](https://circleci.com/gh/qri-io/qri)
-
-#### qri is a global dataset version control system (GDVCS) built on the distributed web
+[![Qri](https://img.shields.io/badge/made%20by-qri-magenta.svg?style=flat-square)](https://qri.io) [![GoDoc](https://godoc.org/github.com/qri-io/qri?status.svg)](http://godoc.org/github.com/qri-io/qri) [![License](https://img.shields.io/github/license/qri-io/qri.svg?style=flat-square)](./LICENSE) [![Codecov](https://img.shields.io/codecov/c/github/qri-io/qri.svg?style=flat-square)](https://codecov.io/gh/qri-io/qri) [![CI](https://img.shields.io/circleci/project/github/qri-io/qri.svg?style=flat-square)](https://circleci.com/gh/qri-io/qri)
+
+<h1 align="center">Qri Backend and CLI</h1>
+
+<div align="center">
+  <img alt="logo" src="https://qri.io/img/blobs/blob_trio.png" width="128">
+</div>
+<div align="center">
+  <strong>a global dataset version control system (GDVCS) built on the distributed web</strong>
+</div>
+
+<div align="center">
+  <h3>
+    <a href="https://qri.io">
+      Website
+    </a>
+    <span> | </span>
+    <a href="#packages">
+      Packages
+    </a>
+    <span> | </span>
+    <a href="https://github.com/qri-io/qri/CONTRIBUTOR.md">
+      Contribute
+    </a>
+    <span> | </span>
+    <a href="https://github.com/qri-io/qri/issues">
+      Issues
+    </a>
+     <span> | </span>
+    <a href="https://qri.io/docs/">
+      Docs
+    </a>
+     <span> | </span>
+    <a href="https://qri.io/download/">
+      Download
+    </a>
+  </h3>
+</div>
+
+<div align="center">
+  <!-- Build Status -->
+</div>
+
+## Welcome
+
+| Question | Answer |
+|--------|-------|
+| "I want to learn about Qri" | [Read the official documentation](https://qri.io/docs/) |
```

Comment:
I secretly want one more row:
**How do I pronounce "Qri"?**, where the answer is obvi "apples".